tests/ui/drop/missing-drop-method.stderr

```diff
@@ -2,9 +2,13 @@ error[E0046]: not all trait items implemented, missing: `drop`
   --> $DIR/missing-drop-method.rs:2:1
    |
 LL | impl Drop for DropNoMethod {}
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `drop` in implementation
```

Comment:
Maybe we can report the old error if `tcx.features().pin_ergonomics()` is not enabled?